README.md

```diff
@@ -7,7 +7,7 @@
 
 Bridge-In-Tech is an application that allows industries/companies, mentors and students to actively collaborate to one another. 
 
-This is the backend client of Bridge-In-Tech.
+This is the backend client of [Bridge-In-Tech](https://github.com/anitab-org/bridge-in-tech-web).
```

Comment:
```suggestion
This is the backend client of [Bridge-In-Tech Web](https://github.com/anitab-org/bridge-in-tech-web).
```

Reply:
> @Aryamanz29 it would be great if we mention that the link points to bridge in tech web

You mean instead of  `Bridge-In-Tech` this right?